src/Database/Type/DateTimeType.php

```diff
@@ -465,13 +465,13 @@ public function getDateTimeClassName(): string
     }
 
     /**
-     * Change the preferred class name to the mutable Time implementation.
+     * Change the preferred class name to the mutable implementation.
      *
      * @return $this
      */
     public function useMutable()
```

Comment:
Shouldn't this method be removed as well?

Reply:
Okay, will remove the the option to use mutable classes altogether. I have kept it as the datettime type classes fallback to `DateTime` and `DateTimeImmuble` is using `cakephp/database` standalone and `cakephp/i18n` isn't loaded.

Reply:
Falling back to `DateTimeImmutable` is always how it has worked though. We'd only be losing the date clamping features when `i18n` isn't present which is how it has historically worked as well.

Reply:
Yip, done.